### ISSUE TITLE
status bar not properly redrawn

### ISSUE THREAD

**Issue by benasocj** [koreader/koreader#11137]
* KOReader version: 2023.10
* Device: Kobo Aura H2O

I have `auto refresh` turned on for the status bar and noticed that the digits for the time are sometimes not rendered correctly. It seems that the area of the digits is not properly redrawn. Notice the digits for the minutes in the following screenshots:
![screenshot1](https://github.com/koreader/koreader/assets/986086/89bb9a24-3312-463a-b755-70007b04ebb0)
![screenshot2](https://github.com/koreader/koreader/assets/986086/35f73a91-a339-4546-8fe3-70e1f0c10f18)
![screenshot3](https://github.com/koreader/koreader/assets/986086/1f13736f-82da-4f26-8653-bc01d1382137)

I recently changed the container height of the status bar to `7` and its font size to `11`. The container bottom margin is set to `1` (that is the default). It seems to me that the problem only started to occur since I made these changes.

**Comment by mergen3107** [comment]
This is a very old issue that no one would like to fix (it's complicated).

See details about Item 1 here: https://github.com/koreader/koreader/issues/9979

For now, I'd suggest using a larger container height.

**Comment by NiLuJe** [comment]
Is this with the default Noto Sans we ship, or did you change the footer font, too?

**Comment by benasocj** [comment]
I only changed the main font to `Noto Serif`. I don't even know how to change the footer font.

**Comment by benasocj** [comment]
Also, I have a gesture in place to enable and disable the status bar. I just noticed that upon disabling the status bar with the gesture, you can clearly see that not all the area in the status bar container area is wiped:
![screenshot2](https://github.com/koreader/koreader/assets/986086/4c0ea1a3-cf27-4df4-89ab-49d4b6ed1907)

Only flipping to the next page redraws the area properly.

On the other hand, upon enabling the status bar again, you can tell that the top area of the status bar is cut off:
![screenshot1](https://github.com/koreader/koreader/assets/986086/3661229e-848b-4f68-a3c8-8e13503e687f)

PS:  I couldn't use the screenshot function to capture this since the screenshot function seems to refresh the screen before it takes the screenshot. That's why I had to take photos of my ebook reader screen which I shared above.

**Comment by mergen3107** [comment]
@benasocj 
All these are very familiar to me :D 
Thanks for your reports, I hope now the issue will get some more attention from devs…

**Comment by NiLuJe** [comment]
The container is not bound by the content (or, put it the other way around, the text is overflowing the container vertically). We have no way to compute that for every possible combination beforehand, so it's not bound at all and the user is left with a footgun at his disposal.

TL;DR:

> For now, I'd suggest using a larger container height.

That. Increase your container height until it doesn't overflow ;).

**Comment by benasocj** [comment]
I reduced the font size which did the trick as well.

Thanks for your help, @mergen3107 and @NiLuJe :blush:.
